CMakeLists.txt

```diff
@@ -168,12 +168,13 @@ write_basic_package_version_file(
 )
 set(CMAKE_SIZEOF_VOID_P ${OLD_CMAKE_SIZEOF_VOID_P})
 
-install(TARGETS range-v3-concepts range-v3-meta range-v3 EXPORT range-v3-targets DESTINATION lib)
-install(EXPORT range-v3-targets FILE range-v3-targets.cmake DESTINATION lib/cmake/range-v3)
+include(GNUInstallDirs)
```

Comment:
What does this do on a Windows system? Or MacOS?

Reply:
Now it will work correctly on all *nixes. On Windows it will be ignored like all install() methods.